crates/optimism/payload/src/payload.rs

```diff
@@ -37,6 +37,8 @@ pub struct OptimismPayloadBuilderAttributes {
     pub transactions: Vec<WithEncoded<TransactionSigned>>,
     /// The gas limit for the generated payload
     pub gas_limit: Option<u64>,
+    /// EIP-1559 parameters for the generated payload
+    pub eip_1559_params: B64,
```

Comment:
this is an option in the rpc payloadattributes, should we also use an option here?